themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -68,6 +68,28 @@
       <?php foreach (($top = $results->getRecommendations('top')) as $current): ?>
         <?=$this->recommend($current)?>
       <?php endforeach; ?>
+      <?php foreach (($side = $results->getRecommendations('side')) as $current):
+        $recommendationClass = str_replace('\\', '_', $current::class);
+        $recommendationJs = <<<JS
+            function addRecommendation_$recommendationClass() {
+              const recommendationContainer = document.querySelector('#search-sidebar .recommendation_container__$recommendationClass');
+              const recommendation = document.getElementById('recommendation__$recommendationClass');
+              recommendationContainer.append(recommendation);
```

Comment:
Is it worth adding any safety checks to be sure recommendationContainer and recommendation actually resolve to something before we start using them? It _should_ be safe to assume they're there, but if something unexpected occurs, I wonder if it would be better to console.error a complaint about a missing element than to have a fatal Javascript error that potentially breaks the whole page.